security/providers/jwt/src/main/java/io/helidon/security/providers/jwt/JwtProvider.java

```diff
@@ -174,7 +174,7 @@ private AuthenticationResponse authenticateToken(String token) {
                 if (validate.isValid()) {
                     return AuthenticationResponse.success(buildSubject(jwt, signedJwt));
                 } else {
-                    return failOrAbstain("Audience is invalid or missing: " + expectedAudience);
+                    return failOrAbstain(errors.toString());
```

Comment:
PLease also modify the comment on line 172, as it is not correct (as `jwt.validate(..)` does more than just validate audience)

Reply:
Done)